meta/core.json

```diff
@@ -31,10 +31,12 @@
         },
         "$defs": {
             "type": "object",
-            "additionalProperties": { "$dynamicRef": "#meta" },
-            "default": {}
+            "additionalProperties": { "$dynamicRef": "#meta" }
         }
     },
+    "default": {
+        "$schema": "https://json-schema.org/draft/2020-12/schema"
```

Comment:
Wouldn't this mean that every schema, including sub-schemas, have a `$schema` keyword. `$schema` has no effect in sub-schemas, so this doesn't make sense to me.

Reply:
Yes, good point. So we'll have to leave it out.. as we have no way currently of specifying in the metaschema "this should apply for the top level schema but not for anything dynamically referenced lower down".